### PR TITLE
Add Klustered episode

### PR DESCRIPTION
**Pull request by JOSHUAJEBARAJ**
This PR add notes for this [episode](https://www.youtube.com/watch?v=51b2imc6OPM)

### CODE REVIEW

File: content/klustered/klustered-klustered-marino-john.hcl [hunk 1]
```diff
@@ -0,0 +1,75 @@
+episode "klustered-klustered-Marino-Wijay-John-Anderson" {
+    title =  "Marino Wijay & John Anderson | Klustered"
+    draft = false 
+    show = "klustered"
+    scheduled_for = "May 12,2022"
```
Comment (rawkode) [inline content/klustered/klustered-klustered-marino-john.hcl:5]: Please use ISO8601 for the date and time

File: content/klustered/klustered-klustered-marino-john.hcl [hunk 2]
```diff
@@ -0,0 +1,75 @@
+episode "klustered-klustered-Marino-Wijay-John-Anderson" {
+    title =  "Marino Wijay & John Anderson | Klustered"
```
Comment (rawkode) [inline content/klustered/klustered-klustered-marino-john.hcl:2]: Remove " | Klustered" from title

File: content/klustered/klustered-klustered-marino-john.hcl [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+  chapter "John Fixing the cluster" {
+    time = "00:52:57"
+  }
+  chapter "Closing remarks" {
+    time = "01:34:46"
+  }
+}
+  description = <<-EOF 
+  SUMMARY
```
Comment (rawkode) [inline content/klustered/klustered-klustered-marino-john.hcl:41]: Line breaks between sections, SUMMARY, LESSONS LEARNED, etc

File: content/klustered/klustered-klustered-marino-john.hcl [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  This episode of Klustered has Guest `Marino Wijay` and `John Anderson` fighting the misconfigured kubecontext, misconfigured kubelet , misconfigured kubescheduler , misconfigured etcd and misconfigured CNI(Cilium)
```
Comment (rawkode) [inline content/klustered/klustered-klustered-marino-john.hcl:42]: Don't wrap guest names in quotes

**Review by rawkode — CHANGES_REQUESTED** [review]

**Comment by JOSHUAJEBARAJ** [comment]
Hey @rawkode  I messed up my commit history I will make a new PR Ignore this one , apologies for the inconvenience

**Pull request closed without merging**